tools/nni_cmd/launcher.py

```diff
@@ -106,7 +106,7 @@ def start_rest_server(port, platform, mode, config_file_name, experiment_id=None
         'You could use \'nnictl create --help\' to get help information' % port)
         exit(1)
     
-    if (platform == 'pai' or platform == 'kubeflow') and detect_port(int(port) + 1):
+    if (platform == 'pai' or platform == 'kubeflow' or platform == 'frameworkcontroller' or platform == 'remote') and detect_port(int(port) + 1):
```

Comment:
Please use platform in [...]

Reply:
updated to in ['local', 'remote', 'pai', 'kubeflow', 'frameworkcontroller']